README.md

```diff
@@ -28,44 +28,59 @@ Look through the application code. If you have the old Lambda Times (Applied Jav
 
 ## Self-Study/Essay Questions
 
-- [ ] What are PropTypes used for? Please describe why it's important to type check our data in JavaScript.
-- [ ] Describe a life-cycle event in React?
-- [ ] Explain the details of a Higher Order Component?
-- [ ] What are three different ways to style components in React? Explain some of the benefits of each.
+- [x] What are PropTypes used for? Please describe why it's important to type check our data in JavaScript.
+
+    Prop Types will type check properties passed to our components.  Prop types protect us from mistakes by ensuring that we are passing the correct data type into as a prop.  It is important to type check to make sure we are passing a component a string when it needs a string, and a number when it needs a number.
+
+- [x] Describe a life-cycle event in React?
+
+    A life-cycle event is a method we call at a certain point in a component's execution that allow us to update the user interface and application states.  For instance during our Mounting Phase we have our Constuctor, componentWillMount(), and render().  Each one performs a specific task at a different point in the life cycle.
+
+- [x] Explain the details of a Higher Order Component?
+
+    A Higher Order Component allow to reuse component logic.  We can pass other components into it as parameters, and process them conditionally such as displaying one component if a condition is met, or displaying another component if another condition is met.
+
```

Comment:
The signature of HOC in React is that it is a function that takes on Component and returns another Component